CHANGELOG.md

```diff
@@ -5,62 +5,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased 2.x]
```

Comment:
I think this needs to be renamed to `Unreleased 2.6`

Reply:
Done - https://github.com/opensearch-project/OpenSearch/pull/6449/commits/959a3026b95940aa2bc95282f3cc710073d524d5